device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/DeviceCalendarPlugin.kt

```diff
@@ -27,7 +27,7 @@ class DeviceCalendarPlugin() : MethodCallHandler {
     val RETRIEVE_CALENDARS_METHOD = "retrieveCalendars";
     val RETRIEVE_EVENTS_METHOD = "retrieveEvents";
     val DELETE_EVENT_METHOD = "deleteEvent";
-    val CREATE_EVENT_METHOD = "createEvent";
+    val CREATE_OR_UPDATE_EVENT_METHOD = "createOrUpdateEvent";
```

Comment:
need to update the request code in the calendar service class as well

Reply:
Yep, forgot about it